gateway/src/apicast/policy_chain.lua

```diff
@@ -96,6 +109,25 @@ function _M:export()
     return chain
 end
 
+--- Freeze the policy chain to prevent modifications.
+-- After calling this method it won't be possible to insert more policies.
+-- @treturn self
```

Comment:
nitpick: missing type

Reply:
👍 

This file is probably not generating the doc as we had a whitelist in config.ld. 

Will add it there and see the generated documentation.

Reply:
Fixed https://github.com/3scale/apicast/pull/496/commits/f4be8fb4f04f5588db901b00b97c1965d40b5078 and going to be squashed.